packages/storybook/stories/CustomIcons/CustomIcons.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { DEFAULT_COMPONENT_ICONS, DEFAULT_COMPOSITE_ICONS } from '@azure/communication-react';
+import { Meta, Canvas, Source } from '@storybook/addon-docs';
+import { IconGridWithSearch } from './IconGrid';
+
+<Meta id="custom-icons" title="Concepts/Custom Icons" />
```

Comment:
I think I would just name and title this "Icons". It talks through both all the default icons, and the customizing icons

Reply:
I changed the titles 10 times at least haha.
If we change it to `Icons` I think the layout of this doc page would need to change as well.